resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -259,32 +260,32 @@ static class FindEntryResult {
     StringPoolRef entry_string_ref;
   }
 
-//  AssetManager2() { memset(&configuration_, 0, sizeof(configuration_)); }
+  //  AssetManager2() { memset(&configuration_, 0, sizeof(configuration_)); }
```

Comment:
Are these formatting changes generated by google-java-format? I remember it will not report error for these existing comments.

Reply:
Yes, changed by the formatter